api/rebac/v1/check.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package api.rebac.v1;
+import "google/api/annotations.proto";
+option go_package = "ciam-rebac/api/rebac/v1;v1";
+option java_multiple_files = true;
+option java_package = "api.rebac.v1";
+
+service Check {
+	rpc CheckPermission (CheckPermissionRequest) returns (CheckPermissionResponse) {
+		option (google.api.http) = {
+			post: "/api/authz/v1/check"
+			body: "*"
+		};
+	};
+}
+
+message CheckPermissionRequest {
+	string subject = 1;
+	string subjectid = 2;
+	string operation = 3;
+	string resourcetype = 4;
+	string resourceid = 5;
+	bool auditlog = 6;
+}
+
+message CheckPermissionResponse {
+	bool result = 1;
+	string description = 3;
```

Comment:
How is this meant to be used? Might be better to leave it out for now if not strongly specified

Reply:
yes, we can remove "description" for now.